manifest: Update LittleFS

Jappie3 opened the pull request:
see https://github.com/zephyrproject-rtos/littlefs/pull/15

Jappie3 commented:
still relevant

de-nordic commented:
> still relevant

@Jappie3 Sorry for lagging. Can you please rebase?

Jappie3 commented:
@de-nordic no worries!
rebased

de-nordic commented:
> @de-nordic no worries!
> 
> rebased

Please check compliance error messages and address issues.

Jappie3 commented:
fixed I think

kartben commented:
> fixed I think

https://github.com/zephyrproject-rtos/zephyr/actions/runs/12845612429/job/35821291559#step:12:46

Jappie3 committed:
manifest: Update LittleFS

The LittleFS module name is now explicitly set in zephyr/module.yml. This
ensures that building with e.g. cmake does not fail when the directory name is
different. When managing dependencies with West, this should be taken care
of automatically.

Signed-off-by: Jappie3 <jasper22034@gmail.com>

Jappie3 commented:
apologies, I assumed signed-off counted as body text
try again

kartben commented:
> apologies, I assumed signed-off counted as body text
> try again

Please do read all the error messages :)
You need to use your legal name for the sign off, as well as git author (and naturally, they must match)

Jappie3 commented:
yeaaaah I don't feel like using my real name on here, sorry guys
thanks for your time
